### ISSUE TITLE
[Request] Portable

### ISSUE THREAD

**Issue by zero77** [butterproject/butter-desktop#685]
Can you please make a portable version that you can run on any desktop without any installation.
Thanks

**Comment by xaiki** [comment]
like a webpage ?

**Comment by zero77** [comment]
Yes, a website/page is one way of doing it and it would be nice to have.

Although, I was meaning something portable so you could download a zip and extract it to a USB and run it on any device without having to install it or even be an admin.

**Comment by xaiki** [comment]
what you suggest already works with a bit of handwork.

**Comment by zero77** [comment]
Is this in the wiki or can you point me to where this is.

Also, you mention a webpage is this also something that is in place or is it being worked on or considered.
